src/android/actions/StartAction.java

```diff
@@ -11,15 +11,27 @@
 public class StartAction implements PushLinkPluginAction {
   private static final String API_KEY = "apiKey";
   private static final String DEVICE_ID = "deviceId";
-  private static final String PACKAGE_NAME = "packageName";
   private static final String ICON = "icon";
 
+  private static final String RESOURCE_DRAWABLE = "drawable";
+  private static final String RESOURCE_MIPMAP = "mipmap";
+  
+
+
   @Override
   public void execute(final CordovaInterface cordova, JSONObject arg, final CallbackContext callbackContext) throws Exception {
-    final String packageName = arg.getString(PACKAGE_NAME);
+    
     final String apiKey = arg.getString(API_KEY);
     final String deviceId = arg.getString(DEVICE_ID);
-    final int appIconId = cordova.getActivity().getResources().getIdentifier(ICON, "drawable", packageName);
+    final String packageName = cordova.getActivity().getApplicationContext().getPackageName();
```

Comment:
Is this a reliable way to get the application package name as it stated in the `AndroidManifest.xml`?